__tests__/conversions.test.js

```diff
@@ -0,0 +1,23 @@
+import { toSeatData } from '../src/conversions';
+
+describe('toSeatData', () => {
```

Comment:
Just a general tip. Usually when unit testing, your descriptions should be more verbose.

In this case, describe could be something like "When using conversions.." (I didn't read your entire project so maybe there's a better description to write than this one). This way, your it functions should complete this sentence.

Your code will then look like this (below) and will be easier to read and understand, especially when unit testing more complex components.

describe("When doing something.."){
   it (' should do this ');
   it (' should do that ');
}

Output: 
When doing something..
  should do this..
  should do that..